accessibility-checker-extension/src/ts/usingAC/UsingACApp.tsx

```diff
@@ -507,6 +504,12 @@ class UsingACApp extends React.Component<{}, UsingACAppState> {
                     available. All views show the same set of issues.
                   </p>
                 </li>
+                <li>
+                  <p style={{ marginTop: "0rem" }}>
+                    To <strong>filter</strong> issues by violations, needs review or by recommendations, press any issue type toggle button in the scan results.
```

Comment:
This might me confusing to people who don't easily recognize 'violations, needs review and recommendations' as the three categories of issues.  Let's spell it out even more clearly, for example:

 "To <strong>filter</strong> issues and focus only on violations, items to review or recommendations, ...."

Also, the phrase 'press any issue type toggle button in the scan results' will be hard to follow, especially if the issue types don't look like a toggle button.  It would help to rephrase the text to something like:

"... select the issue type in the issue count region below the 'Scan' button."

(That way the text includes the name of the region, so a screen reader user can find it easily.)

Going forward this would be a very good place to add an image.